recipe/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "pm4py" %}
+{% set version = "2.7.11.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pm4py-{{ version }}.tar.gz
+  sha256: f655de1c9745546b0f4366c0fdfc635ca6bcdc75381a0701401453c546744a63
+
+build:
+  number: 0
+  # cvxopt not available for s390x and win
+  skip: true  # [s390x or win]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - numpy <2
+    - pandas
+    - deprecation
+    - networkx
+    - python-graphviz
+    - wheel
+    - setuptools
+    - intervaltree
+    - lxml
+    - matplotlib-base
+    - pydotplus
+    - pytz
+    - scipy
+    - tqdm
+    - cvxopt  # [py<313]
+
+test:
+  imports:
+    - pm4py
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pip check
+    - cd tests
+    # as suggested in tests/README.txt
+    - python execute_tests.py
+
+about:
+  home: https://pm4py.fit.fraunhofer.de
+  license: GPL-3.0
```

`GPL-3.0-only` or `GPL-3.0-or-later`?

---

Nothing is specified, so I think it's `GPL-3.0-only`

```suggestion
  license: GPL-3.0-only
```

---

done, thank you for noticing it